src/crypto.rs

```diff
@@ -122,12 +167,14 @@ pub fn fast_aggregate_verify(
     let public_keys = public_keys
         .iter()
         .cloned()
-        .map(bls_impl::PublicKey::try_from)
-        .collect::<Result<Vec<bls_impl::PublicKey>, Error>>()?;
-    let public_keys: Vec<&bls_impl::PublicKey> = public_keys.iter().collect();
-    let signature: bls_impl::Signature = signature.try_into()?;
-    let res = signature.fast_aggregate_verify(true, msg, BLS_DST, &public_keys);
-    if res == BLST_ERROR::BLST_SUCCESS {
+        .map(milagro_bls::PublicKey::try_from)
+        .collect::<Result<Vec<milagro_bls::PublicKey>, Error>>()?;
+    let public_keys: Vec<&milagro_bls::PublicKey> = public_keys.iter().collect();
+    let signature: milagro_bls::Signature = signature.try_into()?;
+    //let res = signature.fast_aggregate_verify(true, msg, BLS_DST, &public_keys);
+    let agg_sig = milagro_bls::AggregateSignature::new();
+    let res = agg_sig.fast_aggregate_verify(msg, &public_keys);
```

Comment:
You are not verifying any signature here.

Reply:
there is the`agg_sig.fast_aggregate_verify(msg, &public_keys);` in line 176

Reply:
Oh, i get what you mean. Will fix it up